### ISSUE TITLE
Error: Cannot find module 'default-browser-id' in v.6.5.2

### ISSUE THREAD

**Issue by muratcorlu** [storybookjs/storybook#18268]
**Describe the bug**

I think #18157 breaks `start-storybook` command.

```
> start-storybook -p 1994

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'default-browser-id'
Require stack:
- /Users/muratcorlu/works/trendyol/grace/node_modules/@aw-web-design/x-default-browser/lib/detect-mac.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@aw-web-design/x-default-browser/index.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/utils/open-in-browser.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/dev-server.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/build-dev.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/index.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core/dist/cjs/server.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core/server.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/web-components/dist/cjs/server/index.js
- /Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/web-components/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/muratcorlu/works/trendyol/grace/node_modules/@aw-web-design/x-default-browser/lib/detect-mac.js:1:25)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@aw-web-design/x-default-browser/lib/detect-mac.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@aw-web-design/x-default-browser/index.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/utils/open-in-browser.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/dev-server.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/build-dev.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core-server/dist/cjs/index.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core/dist/cjs/server.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/core/server.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/web-components/dist/cjs/server/index.js',
    '/Users/muratcorlu/works/trendyol/grace/node_modules/@storybook/web-components/bin/index.js'
  ]
}
```

**To Reproduce**

Sorry `npx sb@next repro` fails on my local as well. But just running to `start-storybook` in v6.5.2 should reproduce this error, I assume.

**System**
```
Environment Info:

  System:
    OS: macOS 12.4
    CPU: (8) arm64 Apple M1
  Binaries:
    Node: 16.15.0 - /usr/local/bin/node
    Yarn: 1.22.10 - ~/.npm/bin/yarn
    npm: 8.5.5 - /usr/local/bin/npm
  Browsers:
    Firefox: 100.0.1
    Safari: 15.5
  npmPackages:
    @storybook/addon-a11y: ^6.5.2 => 6.5.2 
    @storybook/addon-actions: ^6.5.2 => 6.5.2 
    @storybook/addon-essentials: ^6.5.2 => 6.5.2 
    @storybook/addon-links: ^6.5.2 => 6.5.2 
    @storybook/addon-storysource: ^6.5.2 => 6.5.2 
    @storybook/web-components: ^6.5.2 => 6.5.2 
```

**Comment by mgburns** [comment]
Breaks `build-storybook` as well.

**Comment by blackazaru** [comment]
Same error.

**Comment by Chrisdo82** [comment]
same error

**Comment by muratcorlu** [comment]
To see hundreds of "same error" messages here will not help anything and also makes our lives more difficult. Please don't!

**Comment by JonathanSchndr** [comment]
> To see hundreds of "same error" messages here will not help anything and also makes our lives more difficult. Please don't!

ok and my question also does not matter?

**Comment by kitsuned** [comment]
```
yarn add --dev default-browser-id@2.0.0
```

**Comment by muratcorlu** [comment]
I think a new release is already on the way #18270

**Comment by AirmiX** [comment]
Reverting to `6.5.0` worked for me. 
After that I ended up with `Module not found: Error: Can't resolve '@mdx-js/react'` error and deleting generated `Introduction.stories.mdx` file resolved it, since I don't have any other mdx stories for now.

Ofc, it's not the solution, just a temporary workound.

**Comment by muratcorlu** [comment]
> Reverting to `6.5.0` worked for me. After that I ended up with `Module not found: Error: Can't resolve '@mdx-js/react'` error and deleting generated `Introduction.stories.mdx` file resolved it, since I don't have any other mdx stories for now.
> 
> Ofc, it's not the solution, just a temporary workound.

I also reported that mdx issue #18266

**Comment by yannbf** [comment]
Thanks for reporting this issue @muratcorlu.
The issue has been fixed and should be good in `6.5.3`. Please give that a try! Thank you!

**Comment by muratcorlu** [comment]
`default-browser-id` [is only used in macOS](https://github.com/jakub-g/x-default-browser/search?q=default-browser-id). So it's optional for Linux and Windows users but not for macOS users. So if you want to exclude optional packages, you need to install it manually since you are working on a mac.
